sklearn_numba_dpex/common/kernels.py

```diff
@@ -610,39 +579,58 @@ def fused_elementwise_func_(x):
                 f"sub_group_size={sub_group_size} and "
                 f"work_group_size={work_group_size}"
             )
-        check_power_of_2(work_group_size // sub_group_size)
+        n_sub_groups_per_work_group = work_group_size // sub_group_size
+        check_power_of_2(n_sub_groups_per_work_group)
 
     else:
         # Round work_group_size to the maximum smaller power-of-two multiple of
         # `sub_group_size`
         n_sub_groups_per_work_group = get_maximum_power_of_2_smaller_than(
             work_group_size / sub_group_size
         )
+        n_sub_groups_per_work_group = n_sub_groups_per_work_group
         work_group_size = n_sub_groups_per_work_group * sub_group_size
 
+    _is_cpu = device.has_aspect_cpu
```

Comment:
Nit: drop the leading `_` in the variable name if you don't expect a name conflict.

Reply:
I'll revert this `_is_cpu` change, in fact it does not fix the JIT issue. We must wait for the root cause to be addressed in `numba_dpex`